src/pages/Leaderboard/Leaderboard.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { FC } from 'react';
+import React, {
+    FC,
+} from 'react';
```

Comment:
❗️
Что-то странное)
```suggestion
import React, { FC } from 'react';
```